File: reni-frontend/src/main/reni-frontend/src/app/modules/intranet/dto/request/reporte-intervencion-x-usuario-listar-xlsx.request.ts
import { ReporteIntervencionXUsuarioListarXlsxDetRequest } from "./reporte-intervencion-x-usuario-listar-xlsx-det.request";

export class ReporteIntervencionXUsuarioListarXlsxRequest {
    sqlFields: string;
    sqlTables: string;
    listaMapeo: ReporteIntervencionXUsuarioListarXlsxDetRequest[];
    tipDocUsu: string;
    nroDocUsu: string;
    codUsu: string;
}

File: reni-backend/src/main/java/pe/gob/mimp/reni/dto/request/ReporteIntervencionXUsuarioListarXlsxRequest.java
package pe.gob.mimp.reni.dto.request;

import java.io.Serializable;
import java.util.List;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class ReporteIntervencionXUsuarioListarXlsxRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private String sqlFields;
	private String sqlTables;
	private List<ReporteIntervencionXUsuarioListarXlsxDetRequest> listaMapeo;
	private String tipDocUsu;
	private String nroDocUsu;
	private String codUsu;

}